cEpiabm/README.rst

```diff
@@ -0,0 +1,58 @@
+cEpiabm
+=======
+
+We provide an efficient and scalable backend in C++, which can run
+simulations for populations comparable to the UK in a reasonable
+timeframe. This code may be harder than pyEpiabm for new users to
+understand, but the parallels with the python code should be
+sufficiently informative for those who wish to look deeper into the
+code.
+
+Set up
+------
+
+Installation of cEpiabm
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Cmake is used for installation of the cEpiabm software. The following
+procedure will compile the code and run all unit tests:
+
+.. code:: console
+
+   mkdir build_dir
+   cd build_dir
+   cmake ../cEpiabm/. -DCMAKE_BUILD_TYPE=Debug
+   cmake --build . --parallel 2 --target unit_tests
+   ctest -j2 --output-on-failure
+
+Note that cmake must be installed on your system. The following command
+can be used on ubuntu systems:
+
+.. code:: console
+
+   sudo apt-get install cmake cmake-data
+
+We recommend the use of the G++ compiler (version 9 or higher) with
+cEpiabm. If other compilers are used, users should ensure they include 
+the filesystem library, or include this explicitly by inserting 
+`link_libraries(stdc++fs)` in [CMakeLists.txt](cEpiabm/CMakeLists.txt).
```

Comment:
```suggestion
`link_libraries(stdc++fs)` in [CMakeLists.txt](src/CMakeLists.txt).
```

Possible the link is broken because it should be relative to the readme?